source/_docs/blueprint/selectors.markdown

```diff
@@ -737,6 +733,142 @@ entity:
       device_class: motion
 ```
 
+## Floor selector
+
+The floor selector shows a floor finder that can pick a single or multiple
+floors based on the selector configuration. The value of the input will be the
+floor ID, or a list of floor IDs, based on if `multiple` is set to `true`.
+
+A floor selector can filter the list of floors, based on properties of the
+devices and entities that are assigned to the areas on those floors.
+For example, the floor list could be limited to floors with entities
+provided by the [ZHA](/integrations/zha) integration that are on the floor
+based on the area they are in.
```

Comment:
```suggestion
A floor selector can filter the list of floors based on the properties of the
devices and entities assigned to the areas on those floors.
For example, the floor list could be limited to floors with entities
provided by the [ZHA](/integrations/zha) integration and
further limited by the area they are in.
```

Reply:
Tried a rephrase to simplify the sentence a bit. If I misunderstood the meaning, please ignore my comment.

Reply:
It will not "limit further", the area is just the container it searches (as devices & entities are assigned to areas, not floors).

Reply:
oh, I see. I rephrased the rephrase :-D.